src/modules/xcm/tokens/index.ts

```diff
@@ -124,6 +124,26 @@ export const xcmToken = {
       originChain: Chain.BIFROST_POLKADOT,
       minBridgeAmount: '0.1',
     },
+    {
+      symbol: 'EQ',
+      isNativeToken: false,
+      assetId: '18446744073709551628',
+      originAssetId: 'EQ',
+      logo: require('/src/assets/img/EQ.png'),
+      isXcmCompatible: true,
+      originChain: Chain.EQUILIBRIUM,
+      minBridgeAmount: '250',
+    },
+    {
+      symbol: 'EQD',
+      isNativeToken: false,
+      assetId: '18446744073709551629',
+      originAssetId: 'EQD',
+      logo: require('/src/assets/img/EQD.png'),
```

Comment:
This logo looks too dark on dark theme. Could you use replace it to [this](https://resources.acala.network/tokens/EQD.png) file? (save to src/assets/img)

![image](https://user-images.githubusercontent.com/92044428/224217552-0a93c217-51fa-41a8-a980-5f2f07e4639f.png)

Reply:
@pr0fedt you should change the logo of EQD, not EQ. 

https://github.com/AstarNetwork/astar-apps/pull/680/commits/c49ef921682ad5c5a6a92566410c24c4602e8398